pkg/resources/management.cattle.io/v3/clusterroletemplatebinding/validator.go

```diff
@@ -91,6 +92,10 @@ func (a *admitter) Admit(request *admission.Request) (*admissionv1.AdmissionResp
 		if err := validateUpdateFields(oldCRTB, newCRTB, fieldPath); err != nil {
 			return admission.ResponseBadRequest(err.Error()), nil
 		}
+
+		if newCRTB.ClusterName != newCRTB.Namespace {
+			warnings = append(warnings, "Using CRTBs with namespaces and clusterNames that differ is not supported")
```

Comment:
I don't think we need this, because ClusterName and Namespace are both immutable, so this issue would be caught in `validateCreateFields`. If I'm mistaken then we should add the same validation for PRTBs to be consistent.